jobs/forms.py

```diff
@@ -19,3 +19,18 @@ class Meta:
             'title': 'Job title'
         }
 
+
+class MeetupForm(forms.ModelForm):
+
+    class Meta:
+        model = Meetup
+        exclude = ['reviewer', 'review_status', 'reviewers_comment',
+            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        ]
+
```

Comment:
ok, sorry Marysia, I just saw, you have corrected this, but I wrote mine piece before your pull request was merged, so I still got integrity error. Well, I don't know which way is better if any.. Becky, what do you think?